lib/std/unicode.zig

```diff
@@ -538,6 +538,27 @@ fn testUtf8Encode() !void {
     try testing.expect(array[3] == 0b10001000);
 }
 
+test "utf8 encode comptime" {
+    var array: [4]u8 = undefined;
+    array = try utf8EncodeComptime('€');
```

Comment:
You cannot assign `[1]u8` through `[3]u8` to a `[4]u8`, and also `utfEncodeComptime` does not return an error and so `try` cannot be applied.  Instead you can just directly test with `try testing.expectEqualSlices(u8, "€", utf8EncodeComptime('€'));`.

Reply:
shot to soon